geode-core/src/main/java/org/apache/geode/internal/net/SocketCreator.java

```diff
@@ -599,10 +598,6 @@ void configureClientSSLSocket(Socket socket, HostAndPort addr, int timeout) thro
         } else {
           throw ex;
         }
-      } catch (SSLHandshakeException ex) {
```

Comment:
This removal seems like too much to me. The old code treated every SSLHandshakeException as a serious problem by always logging it at the fatal level. I think this may be too much (I'd guess that all the exception handling logging here should be "warn" instead of "fatal") but I don't think you should completely get rid of the logging.
Consider changing this not to log if the exception's message contains "Remote host terminated the handshake" like you do at the higher level.

Reply:
Done

Reply:
I've been thinking about the abundance of error handling in Geode that checks and branches based on brittle exception messages that can change from release to release, vendor to vendor, etc.

We may want to consider doing some searches to see how much code we're talking about and then see what we can do to centralize as much of it as possible so that it's easier to find and figure out what has changed in a JVM release.

Reply:
I'd love to define these error handling branches as behaviors though... such as "what should be done if SSL handshake fails" regardless of what the type and message are.